client/my-sites/plugins-wpcom/standard-plugins-panel.jsx

```diff
@@ -38,6 +45,15 @@ export const StandardPluginsPanel = React.createClass( {
 						) }
 					</div>
 				</CompactCard>
+				<Notice
+					status="is-info"
+					showDismiss={ false }
+					text={ this.translate( 'Uploading your own plugins is not available on WordPress.com.' ) }
```

Comment:
Very minor:
Can we think of plug towards pressable somewhere?
Also it would be cool to mention something like "because of security concerns". Just having a because changes  a lot.